packages/ui/locales/es/views.json

```diff
@@ -1,5 +1,13 @@
 {
   "repos": {
+    "summary": "Summary",
+    "files": "Files",
+    "pipelines": "Pipelines",
+    "commits": "Commits",
+    "pull-requests": "Pull Requests",
+    "webhooks": "Webhooks",
+    "branches": "Branches",
+    "settings": "Settings",
```

Comment:
these should be empty strings?

Reply:
I believe fallbacks get added  to all languages